ecs_fargate/conf.yaml.example

```diff
@@ -3,7 +3,7 @@ init_config:
 
 instances:
   # Change the time to wait on the Fargate metadata API
-  # - timeout: 5
+  - timeout: 5
```

Comment:
I guess what you're meaning by docker friendly, is that we have an instance activated by default when we remove the `.example` right?

Reply:
yup